main.nf

```diff
@@ -111,6 +112,17 @@ if (!params.skip_basecalling) {
             if (!params.kit)        { exit 1, "Please specify a valid kit identifier for basecalling!" }
        }
     }
+    local_model = ""
+    if (params.guppy_model) {
+      if (file(params.guppy_model).exists())   {
+        ch_model = Channel.fromPath(params.guppy_model, checkIfExists: true)
+      } else {
+        ch_model = Channel.empty()
```

Comment:
We could just initialise `ch_model = Channel.empty()` beforehand so we dont need to specify it multiple times afterwards?